contact.js

```diff
@@ -27,6 +42,11 @@ function validateForm() {
       errorMessage.style.display = 'none';
     }, 3000);
   }
+  data.name = fullName.value;
+  data.email = email.value;
+  data.message = textArea.value;
+  const dataString = JSON.stringify(data);
+  localStorage.setItem('data', dataString);
 }
 
 submitBtn.addEventListener('click', validateForm);
```

Comment:
- Kindly, you already implemented storing form data in local storage when submit is cleacked, 👍👍 lets make to save to local storage when any of filed has changed to fit the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md) and hhave better user experience 👍👌